HousingRepairsSchedulingApi.Tests/ControllersTests/AppointmentsControllerTests.cs

```diff
@@ -54,8 +56,9 @@ public async Task TestBookAppointmentEndpoint()
             const string bookingReference = "bookingReference";
             var startDateTime = It.IsAny<DateTime>();
             var endDateTime = It.IsAny<DateTime>();
+            RepairDescription orderComment = new RepairDescription{Text = "something"};
```

Comment:
This duplicates (and hides) the field declared above, I suspect it can be removed.